pay-api/src/pay_api/config.py

```diff
@@ -105,6 +105,7 @@ class _Config():  # pylint: disable=too-few-public-methods
     CFS_ACCOUNT_DESCRIPTION = _get_config('CFS_ACCOUNT_DESCRIPTION', default='BCR')
     CFS_INVOICE_PREFIX = os.getenv('CFS_INVOICE_PREFIX', 'REG')
     CFS_RECEIPT_PREFIX = os.getenv('CFS_RECEIPT_PREFIX', 'RCPT')
+    CFS_PARTY_PREFIX = os.getenv('CFS_RECEIPT_PREFIX', '')
```

Comment:
need fix on the env var name. Also better to default it to BCR? But check with John or Jyoti first.